lib/cgraph/edge.c

```diff
@@ -498,11 +498,27 @@ and to expose them to foreign languages without C preprocessor. */
 #ifdef ageqedge
 #undef ageqedge
 #endif
-CGRAPH_API Agnode_t *ageqedge(Agedge_t * e, Agedge_t * f)
+CGRAPH_API int ageqedge(Agedge_t * e, Agedge_t * f)
```

Comment:
Why is the return type of this function changed to `int`?

Reply:
It should have been that before, as the return type of ageqedge is boolean. If I'm correct, the `int` type is used to expose a boolean value. A carelessness of my side in an earlier pull request.